torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2498,7 +2542,7 @@ Tensor linalg_qr_backward(const std::vector<torch::autograd::Variable> &grads, c
 }
 
 // Invertible case is derived from Jacobi's formula, and also can be found at:
-// http://eprints.maths.ox.ac.uk/1079/1/NA-08-01.pdf
+// https://people.maths.ox.ac.uk/gilesm/files/NA-08-01.pdf
```

Comment:
A leak from a different branch? :)

Reply:
I corrected the old URL (which does not work) to one that works.

Reply:
It's just these two. That being said, I'm going to revert this one, as we are upgrading the derivative for the determinant soon anyway. We can change it then really.